facade/interface.go

```diff
@@ -34,6 +34,7 @@ type NodeHandler interface {
 
 	//ValidateTransaction will validate a transaction
 	ValidateTransaction(tx *transaction.Transaction) error
+	ValidateTransactionForTxSimulation(tx *transaction.Transaction) error
```

Comment:
maybe rename to `ValidateSimulatedTransaction`? or `ValidateTransactionForSimulation` ?

Reply:
done